### ISSUE TITLE
kubectl create pod error -  Kubernetes cluster on AWS using cloudformation template

### ISSUE THREAD

**Issue by santhoshdaivajna** [GoogleCloudPlatform/kubernetes#10220]
Hello,

I ran a two node kubernetes cluster on AWS with coreos through the cloudformation template.  I am hitting a road block while creating a pod. Please check if this is related to kubectl/kubernetes version error.

I followed the instruction in this doc. [link to doc!](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/aws-coreos.md)
```
ip-172-20-0-254 ~ # kubectl get nodes
NAME           LABELS                                STATUS
172.20.0.249   kubernetes.io/hostname=172.20.0.249   Ready
172.20.0.250   kubernetes.io/hostname=172.20.0.250   Ready

ip-172-20-0-254 ~ # kubectl get pods
POD       IP        CONTAINER(S)   IMAGE(S)   HOST      LABELS    STATUS    CREATED   MESSAGE

ip-172-20-0-254 ~ # kubectl version
Client Version: version.Info{Major:"0", Minor:"18", GitVersion:"v0.18.2", GitCommit:"1f12b893876ad6c41396222693e37061f6e80fe1", GitTreeState:"clean"}
Server Version: version.Info{Major:"0", Minor:"18", GitVersion:"v0.18.2", GitCommit:"1f12b893876ad6c41396222693e37061f6e80fe1", GitTreeState:"clean"}

ip-172-20-0-254 ~ # kubectl create -f pod.json
Error from server: the server could not find the requested resource
ip-172-20-0-254 ~ #
```
-Santhosh

**Comment by santhoshdaivajna** [comment]
works with kubernetes 0.19.0 
changed all refereneces of 0.18.2 in the cloudformation template to 0.19.0

kubectl create -f pod.json   # is working fine.

**Comment by jlowdermilk** [comment]
```
ip-172-20-0-254 ~ # kubectl create -f pod.json
Error from server: the server could not find the requested resource
```
That error is caused by client/server incompatibility. It's a little disturbing that you saw it with the same client/server version but the issue should be resolved for 0.19.0 or greater clients.
